### DISCUSSION TITLE
URL visited, list of visitor IPs

### DISCUSSION THREAD

**Discussion by yaonetuno** [allinurl/goaccess#2396] [Q&A]
Hello,

Sorry if I am duplicating this question, I tried to look for it but did not find it.
I would like to know if it is possible by clicking the URL, it will give the list of all the visitor's IP?
Thank you in advance for your attention.

Best regards,
Wei Yao

**Comment by allinurl** [comment]
Great question. **#117** will address this in a nice way. I’m working on it as we speak. [Stay tuned!](https://twitter.com/goaccess)
